components/microsoft_teams/microsoft_teams.app.mjs

```diff
@@ -5,6 +5,7 @@ import constants from "./common/constants.mjs";
 export default {
   type: "app",
   app: "microsoft_teams",
+  description: "**Personal account is not currently supported by Microsoft Teams.** [see more information here](https://learn.microsoft.com/en-us/graph/permissions-reference#remarks-7).",
```

Comment:
@alysonfreitas can you update to,

description: "**Personal accounts are not currently supported by Microsoft Teams.** Refer to Microsoft's documentation [here](https://learn.microsoft.com/en-us/graph/permissions-reference#remarks-7) to learn more.",

Reply:
@dannyroosevelt attaching the PR with the changes required: https://github.com/PipedreamHQ/pipedream/pull/5062